gui/wxpython/datacatalog/catalog.py

```diff
@@ -110,20 +110,26 @@ def OnCreateMapset(self, event):
         db_node, loc_node, mapset_node = self.tree.GetCurrentDbLocationMapsetNode()
         self.tree.CreateMapset(db_node, loc_node)
 
-    def OnImportOgrLayers(self, event, cmd=None):
```

Comment:
Do we need `Import*Layers()`? What about:

```
    def OnImportOgrLayers(self, event):
        from modules.import_export import OgrImportDialog
        dlg = OgrImportDialog(parent=self, giface=self.giface)
        dlg.CentreOnScreen()
        dlg.Show()
```

Reply:
If we called it later in frame.py it would make sense to have only ImportOgrLayers however as we decided to not connect Info Bars to File menu events, we do not need it...